### DISCUSSION TITLE
how to enable ubo in family link for my kid's account

### DISCUSSION THREAD

**Discussion by dnsmo** [uBlockOrigin/uBlock-issues#2857] [Q&A]
The extension was able to install fine but I just can't enable it. Does anyone have a similar problems? I did a search and can't find anything.

**Comment by gothic-bum** [comment]
uBO is a browser extension. It doesn't work in apps.

**Reply by dnsmo** [reply]
I am using google chrome to install it. not talking about the apps.

**Reply by gothic-bum** [reply]
Then why did you mention "family link", which is an app?
  
What is the specific issue you are having with uBO?

**Reply by dnsmo** [reply]
family link is not just an app but a service provided by google which I used to help monitor my kids online activity. Then I want to use UBO under her account so it will help block some harmfully ADs, she has been misclicking on them on kid's websites and keep installing bad application to the PC. Thanks

**Reply by gothic-bum** [reply]
uBO is a browser extension and not an app, and cannot be managed with Family Link.
  
For information about what Family Link can manage, see:  
https://support.google.com/families/topic/7336731

**Reply by dnsmo** [reply]
thank you for the reply but I already check them all out, I was set it allow on the chrome, it installed the extension fine, just not I can't enable it.

**Reply by Yuki2718** [reply]
Have you read `https://support.google.com/chromebook/answer/9900680`?

**Reply by krystian3w** [reply]
Should works for Progressive apps: https://en.wikipedia.org/wiki/Progressive_web_app
